bin/sphx_ex2nb.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+r"""
+Sphinx Gallery Notebook converter
+=================================
+
+Exposes the Sphinx-Gallery Notebook rederer to directly convert Python
+scripts into Jupyter Notebooks.
+
+"""
+# Author: Óscar Nájera
+# License: 3-clause BSD
+
+from __future__ import division, absolute_import, print_function
+import argparse
+import os
+
+import sphinx_gallery.gen_rst as sg
+from sphinx_gallery.notebook import Notebook
+
+parser = argparse.ArgumentParser(
+    description='Sphinx-Gallery Notebook converter')
+parser.add_argument('file', nargs='+',
+                    help='Input Python file script to convert. '
+                    'Supports multiple files and shell wildcards'
+                    ' (e.g. *.py)')
+
+
+def main():
+    args = parser.parse_args()
+
+    for src_file in args.file:
+        file_name = os.path.basename(src_file)
```

Comment:
For consistency I think this should be `src_filename` and `filename`. I tend to use `filename` for file names and `file` for object files (i.e. as returned from `open`) to make it easier to differentiate between the two.